gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +55,16 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
+if [ -d genconf/ca ]; then
```

Comment:
Nit: Would be good to have a code comment here in which case `genconf/ca` is expected to be present, and in which case it's not expected to be present.